newapi/ooniapi/prio.py

```diff
@@ -37,17 +37,33 @@
 prio_bp = Blueprint("prio", "probe_services_prio")
 
 
-# # failover algorithm
+# # fallback algorithm
 
 CTZ = namedtuple("CTZ", ["url", "category_code"])
-failover_test_items: Dict[str, List[CTZ]] = {}
+fallback_test_items: Dict[str, List[CTZ]] = {}
 
 
-def failover_fetch_citizenlab_data() -> Dict[str, List[CTZ]]:
+def init_fallback_test_list(log, app):
     """Fetches the citizenlab table from the database.
-    Used only once at startime for failover."""
-    log = current_app.logger
-    log.info("Started failover_fetch_citizenlab_data")
+    Used only once at startime for fallback."""
+    global fallback_test_items
+    with app.app_context():
+        log = current_app.logger
+        log.info("initializing fallback test-list")
+        try:
+            fallback_test_items = fallback_fetch_citizenlab_data()
+        except Exception:
+            log.error("fallback test-list init error", exc_info=True)
```

Comment:
Should we perhaps log the metric as zero in this case?

Reply:
Incremental counters would not change with a value of 0 although they might record the event perhaps. I'm not sure how that would be useful.

Reply:
I was referring to the metrics.incr() call. The test list size is indeed a gauge and the value will be 0 if the test list is empty.